### ISSUE TITLE
Engagement Experience Dashboards - APEX Program, Releases, TypeSpec milestone/tasks interval tracking

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#8226]
Creation of PowerBI Dashboards for APEX Program and Release Plans.  

Release Plan and APEX Prototype is in progress - https://msit.powerbi.com/groups/3e17dcb0-4257-4a30-b843-77f47f1d4121/reports/75926d99-0ce6-430e-87f5-684fbc90958b/5c677b3eacf6dca228fa?ctid=72f988bf-86f1-41af-91ab-2d7cd011db47&experience=power-bi

Deliverables: 
- [x] APEX Program Dashboard Prototype (June/July)

```[tasklist]
### General TODOs
- [x] Send email with link to prototype PowerBI Report to Josephine.  Requested that she experiment with using the Release page and that I would schedule time with her next week to discuss what would be useful for management plane page/customized visualizations.
- [x] Schedule meeting with Josephine 1H June to discuss management plane release plan page (insights/metrics).
- [x] Schedule meeting with Mario 1H June to look over visuals and data in the TypeSpec (time spent on tasks in release planner) PowerBI report.
```

```[tasklist]
### Release tracking dashboards/visuals
- [ ] Customized views/dashboards for data & management release tracking
- [x] Gaps that need to be addressed for DPG stakeholder insights/metrics - https://github.com/Azure/azure-sdk-tools/issues/8194
```
```[tasklist]
###  All Release page
- [x] Release Plan Dashboard Stakeholder Feedback with Renhe and Laurent scheduled for 05/15/2024.
- [ ] Gaps in data need to be resolved - https://github.com/Azure/azure-sdk-tools/issues/8225.
- [ ] % of Self-Service release plans (Release page):  Show % release plans created by service partners vs created by SDK team (release plan submitted by field will not contain "ladonnaquinn@microsoft.com or "v-yoganandak@microsoft.com".  @AngelosP
```

```[tasklist]
### APEX page
- [x] Add the following filter options to the APEX page - RP Name, Product Type, PrivPrev State, PubPrev State, GA State
- [x] Add visuals to APEX page that show Product types, PrivPrev State, PubPrev State, GA State.  @AngelosP
- [x] Confirm that APEX page does not include product epics (epic type=product) that State = New, Duplicate, Closing Down, Retired and Tags do not contain "Release Planner App Test" or "non-APEX tracking".  @AngelosP
```

```[tasklist]
### Onboarding page
- [ ] New page named Onboarding. 
- [ ] **Product Onboarding visual (bar graph)** Bar graph that shows total number of products onboarded each month.  Triage work items grouped by the Product Onboard Date date field. Triage work items for products will have data in the Product Service Tree ID field.  If data value is blank or N/A, the triage work item should be counted as service onboarding. @AngelosP 
- [ ]  **Product Onboarding visual (from bar graph data)**. Visual that shows product types. @AngelosP 
- [ ] **Product Onboarding table (show below bar graph**).  Fields to display: Link to triage work item, link to related epic (product type), State field (name Onboard State in table), Product Name, Product Service Tree ID, Product Type, Info Meeting State, Info Meeting Date @AngelosP 
- [ ] Product Onboarding filter options for bar graph, table, ...).  Filter by State, DataScope, MgmtScope, Info Meeting State, Product Service Tree ID, Associated Service ServiceTree ID @AngelosP 
- [ ] **Service Onboarding visual (bar graph):**  Bar graph that shows number of services onboarded (no product).  There will be no data or N/A in the Product Service Tree ID ADO data field of triage work item.  @AngelosP 
- [ ] **Service Onboarding visual for product type**. Visual that shows product types for services onboarded (no data or N/A in the Product Service Tree ID triage work item field)  @AngelosP 
- [ ] **Service Onboarding table.**  Fields to display: Link to triage work item, link to related epic (product type), State field (name Onboard State in table), Product Name, Product Service Tree  ID, Product Type, Info Meeting State, Info Meeting Date @AngelosP 
- [ ] Service Onboarding filters: Link to triage work item, link to related epic (product type), State field (name Onboard State in table), Product Name, Product Service Tree ID, Product Type, Info Meeting State, Info Meeting Date @AngelosP 
- [ ]  **% of Self-Service products onboarded** (Onboarding page) Visual that indicates the % and number that are self-service.  Logic to determine self-service onboarding - IF Triage work item (Onboarding Questionnaire Submitted by ADO field <> "Yogananda Krishnaswamy" OR "LaDonna Quinn") AND (Info Meeting State= Not Required" OR "Cancelled" OR field is blank), THEN the product was onboarded via self-service.  @AngelosP 
- [ ] **% of Self-Service services onboarded** (Onboarding page). Visual that indicates the % and number that are self-service.  Logic to determine self-service onboarding - IF Triage work item (Onboarding Questionnaire Submitted by ADO field <> "Yogananda Krishnaswamy" OR "LaDonna Quinn") AND (Info Meeting State= Not Required" OR "Cancelled" OR field is blank), THEN the product was onboarded via self-service.  @AngelosP 
- [ ] Clean up the Product Onboard Data data in Azure Devops.  Triage work items migrated or created in 2023 do not have any data in the Product Onboard data field. Will need to export to excel and update manually.  The Product Onboard Date will be the "Created date" in the archived onboard work item.  @ladonnaq 
```

**Comment by ladonnaq** [comment]
No further progress can be made until engineering completes the back-end work to add or expose new data fields for the dashboard.  Once this work is done, then we will resume activities on progressing the dashboard.

**Comment by MrJustinB** [comment]
Closing this is not a priority now. We can reopen if we determined it is a priority in the future
